code/datums/mob_hud.dm

```diff
@@ -348,6 +348,8 @@ var/list/datum/mob_hud/huds = list(
 	holder4.icon_state = "hudblank"
 
 	if(species && species.flags & IS_SYNTHETIC)
+		if(HAS_TRAIT(src, TRAIT_MASKED_BIO))
```

Comment:
this isn't generic enough, as this only applies to synths while being a generic looking trait. additionally, i don't think this makes "hudhealthy" show up, which automatically looks suspicious. you probably need to write some custom code for that